### ISSUE TITLE
Does not appear to work with *.yml files inside node_modules

### ISSUE THREAD

**Issue by trajano** [wyattis/babel-plugin-content-transformer#1]
It works well for YAML files in a project, but if I want to load up the same YAML files when I packed it into a module, it does not load it correctly, it just ends up as numbers.

**Comment by wyattis** [comment]
What does your config look like?

**Comment by trajano** [comment]
```
module.exports = function (api) {
  api.cache(true);
  return {
    presets: ['babel-preset-expo'],
    plugins: [
      [
        'content-transformer',
        {
          transformers: [
            {
              file: /\.ya?ml$/,
              format: 'yaml',
            },
          ],
        },
      ],
  };
};
```

I tried to put it in a module, it's the same as I have in my app code except in my app I add `dotenv-import`.

For now I am using `yaml.macro` to get around the problem.  But ideally I am trying to see if there's some sort of content transformer so that when the babel runs on the YAML it will output JS code.

**Comment by wyattis** [comment]
Do you have a public npm module with yaml files that I could test with?

**Comment by trajano** [comment]
No it's an internal project, an MVCE  would have to be created to show this problem.  The MVCE would just be...
1. create a react-native library using `bob` and add a YAML file and set up the content-loader.  The yaml file is exposed in `index.js` as an export
2. create an expo project that loads from the other module and try to access the YAML flie from the index.

**Comment by wyattis** [comment]
Thanks for filing this issue. It should be fixed in v0.2.0, but if it isn't, please reopen this issue.
